src/test/java/uk/gov/hmcts/cmc/claimstore/services/FormaliseResponseAcceptanceServiceTest.java

```diff
@@ -463,7 +496,7 @@ public void formaliseDoesNothingWhenReferredToJudge() {
     }
 
     @Test
-    public void formaliseDoesNothingWhenResponseIsNotAcceptation() {
+    public void doNotFormaliseWhenResponseIsNotAcceptation() {
```

Comment:
Not part of this story but if formalise is not possible upon rejection then no point accepting both subtypes of `ClaimantResponse` as an argument.